test/quantization/fx/test_quantize_fx.py

```diff
@@ -4082,6 +4082,57 @@ def forward(self, inputs: torch.Tensor, state: List[torch.Tensor]):
             if n.target == "lstm":
                 self.assertEqual(type(n.args[1]), tuple)
 
+    def test_static_lstm(self):
```

Comment:
I think we can add another test that uses lstm in the middle of the model and check if it works with surrounding operators or not